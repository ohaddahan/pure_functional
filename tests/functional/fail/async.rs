use functional_macro::pure_functional;

#[pure_functional]
async fn test() {}

fn main() {
    test();
}
