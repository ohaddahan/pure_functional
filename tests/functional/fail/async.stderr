error: pure_functional: attribute on async fn is not supported
 --> tests/functional/fail/async.rs:3:1
  |
3 | #[pure_functional]
  | ^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `pure_functional` (in Nightly builds, run with -Z macro-backtrace for more info)
